src/client/eppo-precomputed-client.ts

```diff
@@ -237,6 +237,7 @@ export default class EppoPrecomputedClient {
       allocationKey: precomputedFlag.allocationKey ?? '',
       extraLogging: precomputedFlag.extraLogging ?? {},
       doLog: precomputedFlag.doLog,
+      entityId: null,
```

Comment:
Is this something that needs to be returned from the assignments endpoint so that the precomputed client has it available as well?